docs/tools/prepare_examples_tests.py

```diff
@@ -11,7 +11,7 @@
 
 # settings
 SKIP_FOLDERS = ["archive", ".", "_", "local_cache"]
-SKIP_EXAMPLES: List[str] = []
+SKIP_EXAMPLES: List[str] = ["custom_destination_lancedb"]
```

Comment:
this was needed because the script was just hanging up while trying to connect to the local lancedb (spotify.db)
![image](https://github.com/user-attachments/assets/1c6f5fa3-f53b-4650-aecf-734122acfb47)
so the lint-and-test-exapmle job would not finish but keep on running